_FIPS201/requirements.md

```diff
@@ -495,7 +495,7 @@ can be reset in certain circumstances.
 
 The PIN on a PIV Card may need to be reset if the cardholder has
 forgotten the PIN or if PIN-based cardholder authentication has been disabled by the usage of an
-invalid PIN more than the allowed number of retries. A maximum of 10 consecutive PIN retries **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
+invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has developed epidermal scaring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries (i.e., PIN or OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
```

Comment:
```suggestion
invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has developed epidermal scaring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries for each of the activation mechanism (i.e., PIN and OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
```

Reply:
Yep. Adding an 's' to methods and will add it.